packages/block-editor/src/components/grid-visualizer/grid-item-resizer.js

```diff
@@ -55,26 +157,32 @@ export function GridItemResizer( { clientId, onChange } ) {
 					);
 					const columnStart = getClosestLine(
 						gridColumnLines,
-						blockElement.offsetLeft
+						blockElement.offsetLeft + boxElement.offsetLeft
```

Comment:
In https://github.com/WordPress/gutenberg/pull/59490 I changed the approach to use _tracks_ instead of _lines_ ([see here](https://github.com/WordPress/gutenberg/pull/59490/files#diff-87c248a2915bbac0f7dca7641cbc344e5cf0598b82000f40a09cbb5535bc603bR68)) which, aside from being a more resilient approach, I _think_ should make this not necessary.

Want me to push to this PR? Or open a new one?

Reply:
Hmm my inclination is push to this PR if it's a smallish change, and do a separate one if it's a big one, but not too fussed either way!

Reply:
Opened a PR here https://github.com/WordPress/gutenberg/pull/61423

Apologies for not doing this earlier!

Reply:
If you rebase to include https://github.com/WordPress/gutenberg/pull/61423 is this still necessary?

Reply:
Didn't quite fix it. Looking into it now

Reply:
Uh actually this is completely borked now, and I'm having trouble working out why 😬 
The problem is it's assuming resizes are only possible to the bottom/right, so it doesn't let blocks on the right or bottom edges grow at all.

Reply:
I pushed what I think is a fixed... more or less what you already had with combining `boxElement` and `blockElement`'s rects 😅

Reply:
Yup, that's working well for me, thanks for fixing it!